CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 
 - Redirect endpoint to receive TikTok's redirect
+- Middleware for parsing AuthCode and state params
```

Comment:
This should be `## [Unreleased]`, shouldn't it?
```suggestion
## [Unreleased]

### Added

- Middleware for parsing AuthCode and state params
```

Reply:
Done